src/main/java/reposense/model/RepoLocation.java

```diff
@@ -41,6 +41,10 @@ public RepoLocation(String location) throws InvalidLocationException {
         }
     }
 
+    public boolean isEmpty() {
```

Comment:
`isEmpty()` is too vague. It gives the impression that it is testing `location` or `RepoLocation`'s attributes, but in actual only `repoName` is being tested.

Reply:
This is a bug. I have fixed it.